apps/web/ee/components/support/HelpMenuItem.tsx

```diff
@@ -1,13 +1,213 @@
-import HelpscoutMenuItem from "@ee/lib/helpscout/HelpscoutMenuItem";
-import IntercomMenuItem from "@ee/lib/intercom/IntercomMenuItem";
-import ZendeskMenuItem from "@ee/lib/zendesk/ZendeskMenuItem";
+import { ExternalLinkIcon, ExclamationIcon } from "@heroicons/react/solid";
+import { useState } from "react";
+
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import Button from "@calcom/ui/Button";
+
+import classNames from "@lib/classNames";
+
+import ContactMenuItem from "./ContactMenuItem";
 
 export default function HelpMenuItem() {
```

Comment:
❓ Shouldn't this work through tRPC

Reply:
Agreed @emrysal, this will reduce the code significantly, extra points for actually sticking to React Hook Forms.